src/components/Reports/ReportsBox/ReportsBox.test.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import DopplerIntlProvider from '../../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import ReportsBox from './ReportsBox';
+import { addDays } from '../../../utils';
+
+describe('ReportsBox component', () => {
+  afterEach(cleanup);
+
+  it('should show loading when flag is set to loading and visits are null or undefined', () => {
+    // Arrange
+    const today = new Date();
+    const dateFrom = addDays(today, -1);
+    const dateTo = today;
+    const visits = null;
+
+    // Act
+    const { container } = render(
+      <DopplerIntlProvider>
+        <ReportsBox
+          dateFrom={dateFrom}
+          dateTo={dateTo}
+          emailFilter={'with_email'}
+          today={today}
+          visits={visits}
+        />
```

Comment:
You have updated the test name, but I do not see the flag :P

Reply:
i put now is more clear